### DISCUSSION TITLE
C&W attack - Max_Iter

### DISCUSSION THREAD

**Discussion by MiguelCosta94** [Trusted-AI/adversarial-robustness-toolbox#1741] [Q&A]
When exploring the Carling and Wagner attack I found that your implementation is slightly different from the original. Does your implementation require fewer steps for the gradient descent than the original implementation? I see the original implementation takes a default value of 1000 and yours only 10.

**Comment by beat-buesser** [comment]
Hi @MiguelCosta94 That's a good question, I don't think that our implementation needs less iterations, but rather that for some reason the default value has been chosen to be lower. Are you able to achieve comparable attack success rates if you increase the number of iterations?

I convert this issue into a Discussion for now.

**Reply by MiguelCosta94** [reply]
Hi @beat-buesser I will increase the number of iterations and I will report the results. It may take some time (days to weeks) due to hardware limitations.
